src/app/forms/components/defect-select/defect-select.component.scss

```diff
@@ -49,40 +57,208 @@
-      display: inline-flex;
-      color: inherit;
-      justify-content: space-between;
-      min-width: 150px;
-      text-align: left;
-      padding: 15px 30px 20px 15px;
-      width: 100%;
-    }
+.browse__link {
+  font-family: 'GDS Transport', arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-decoration: underline;
+  text-decoration-thickness: max(1px, 0.0625rem);
+  text-underline-offset: 0.1em;
+  font-family: 'GDS Transport', arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1rem;
+  line-height: 1.25;
+  position: relative;
+  display: block;
+  padding: 15px 25px 20px 0;
+  color: #1d70b8;
+}
 
-    &--more {
-      padding-left: 1rem;
-      &::after {
-        content: '›';
-        color: #000000;
-      }
-    }
+.browse__link:hover {
+  text-decoration-thickness: max(3px, 0.1875rem, 0.12em);
+  -webkit-text-decoration-skip-ink: none;
+  text-decoration-skip-ink: none;
+  -webkit-text-decoration-skip: none;
+  text-decoration-skip: none;
+}
+
+.browse__link:focus {
+  outline: 3px solid transparent;
+  color: #0b0c0c;
+  background-color: #fd0;
+  box-shadow: 0 -2px #fd0, 0 4px #0b0c0c;
+  text-decoration: none;
+}
+
+.browse__link:link {
+  color: #1d70b8;
+}
+
+.browse__link:visited {
+  color: #1d70b8;
+}
+
+.browse__link:hover {
+  color: #003078;
+}
+
+.browse__link:active {
+  color: #0b0c0c;
+}
+
+.browse__link:focus {
+  color: #0b0c0c;
+}
+
+@media (min-width: 40.0625em) {
+  .browse__link {
+    font-size: 19px;
+    font-size: 1.1875rem;
+    line-height: 1.3157894737;
   }
 }
+@media print {
+  .browse__link {
+    font-size: 14pt;
+    line-height: 1.15;
+  }
+}
+@media (min-width: 40.0625em) {
+  .browse__link {
+    padding: 15px 40px 20px 15px;
+    font-family: 'GDS Transport', arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    font-weight: 400;
+    font-size: 14px;
+    font-size: 0.875rem;
+    line-height: 1.1428571429;
+  }
+}
+@media print and (min-width: 40.0625em) {
+  .browse__link {
+    font-family: sans-serif;
+  }
+}
+@media (min-width: 40.0625em) and (min-width: 40.0625em) {
+  .browse__link {
+    font-size: 16px;
+    font-size: 1rem;
+    line-height: 1.25;
+  }
+}
+@media print and (min-width: 40.0625em) {
+  .browse__link {
+    font-size: 14pt;
+    line-height: 1.2;
+  }
+}
+.browse__link:hover {
+  background: #f3f2f1;
+  color: #1d70b8;
+}
+
+.browse__link:focus:hover {
+  box-shadow: 0 -2px #f3f2f1, 0 3px #0b0c0c;
+  color: #0b0c0c;
+}
+
+.browse__link:after {
+  position: absolute;
+  top: 50%;
+  margin-top: -8px;
+  right: 10px;
+  float: right;
+  content: '›';
+}
+
+.browse__link.loading {
+  cursor: default;
+  background: transparent url(/assets/collections/loading-8291cf1cf2baabb70cbcff73ecbfdb8012a9c05c619b020c8921c0fcd6ef389a.gif) 95% 50% no-repeat;
```

Comment:
Not sure what this is, however there is nothing in `assets/collections`. also unsure why we would use a gif for the loading spinner if we have a component that does that already